bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,212 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
-## Script Examples
+The following examples show how to access common openHAB functionalities.
+
+## Logging
 
-JavaScript scripts provide access to almost all the functionality in an openHAB runtime environment.
 As a simple example, the following script logs "Hello, World!".
 Note that `console.log` will usually not work since the output has no terminal to display the text.
 The openHAB server uses the [SLF4J](https://www.slf4j.org/) library for logging.
 
-```js
-const LoggerFactory = Java.type('org.slf4j.LoggerFactory');
-
-LoggerFactory.getLogger("org.openhab.core.automation.examples").info("Hello world!");
+```javascript
+let logger = Java.type('org.slf4j.LoggerFactory').getLogger('org.openhab.rule.' + this.ruleUID);
+logger.info('Hello world!');
+logger.warn('Successfully logged warning.');
+logger.error('Successfully logged error.');
 ```
 
-Depending on the openHAB logging configuration, you may need to prefix logger names with `org.openhab.core.automation` for them to show up in the log file (or you modify the logging configuration).
-
 The script uses the [LoggerFactory](https://www.slf4j.org/apidocs/org/slf4j/Logger.html) to obtain a named logger and then logs a message like:
```

Comment:
Note: we don't really want to encourage getting LogManagers directly, but should prefer going through a helper library. This is fine until that is available, however.

Reply:
Okay, should I add an extra note here?